app/helpers/users_helper.rb

```diff
@@ -29,8 +29,9 @@ def options_for_partners
     policy_scope(Partner).all.order(:name).pluck(:name, :id)
   end
 
-  def options_for_neighbourhoods
+  def options_for_user_neighbourhoods
     policy_scope(Neighbourhood)
+      .latest_release
       .where('name is not null and name != \'\'')
```

Comment:
just a question for my own understanding. Is there a reason we wouldn't want this as part of the scope?

Reply:
This constrains the front-end to only find the latest neighbourhoods for the neighbourhood pickers